libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -805,6 +806,36 @@ void YamlConfiguration::parsePropertiesNodeYaml(const YAML::Node& propertiesNode
   validateComponentProperties(processor, component_name, yaml_section);
 }
 
+void YamlConfiguration::parseFunnelsYaml(const YAML::Node& node, core::ProcessGroup* parent) {
+  if (!parent) {
+    logger_->log_error("parseFunnelsYaml: no parent group was provided");
+    return;
+  }
+  if (!node || !node.IsSequence()) {
+    return;
+  }
+
+  for (const auto& element : node) {
+    YAML::Node funnel_node = element.as<YAML::Node>();
+
+    std::string id = getOrGenerateId(funnel_node);
+
+    // Default name to be same as ID
+    std::string name = funnel_node["name"].as<std::string>(id);
+
+    const utils::optional<utils::Identifier> uuid = utils::Identifier::parse(id) | utils::orElse([this] {
+      logger_->log_debug("Incorrect connection UUID format.");
+      throw Exception(ExceptionType::GENERAL_EXCEPTION, "Incorrect connection UUID format.");
```

Comment:
both of these should be "Incorrect funnel UUID format."

Reply:
Updated in e64b1c0d1fe231936dcde263939fbd8dcfff4083